astroplan/constraints.py

```diff
@@ -506,15 +506,35 @@ def is_always_observable(constraints, observer, targets, times=None,
         List of booleans of same length as ``targets`` for whether or not each
         target is observable in the time range given the constraints.
     """
+    targets_is_scalar = False
+    times_is_scalar = False
+
+    if not hasattr(targets, '__len__'):
```

Comment:
As with `Time` below, checking for `__len__` may not work if a `SkyCoord` is given, because it has a `__len__` but it raises an error when used.  (If you're wondering, this behavior is also true of numpy arrays...).  So maybe ``not hasattr(targets, '__len__') or (hasattr(targets, 'isscalar') and targets.isscalar)``?